agent/grpc-external/services/peerstream/stream_test.go

```diff
@@ -905,10 +940,13 @@ func TestStreamResources_Server_TerminatesOnHeartbeatTimeout(t *testing.T) {
 	})
 
 	testutil.RunStep(t, "stream is disconnected due to heartbeat timeout", func(t *testing.T) {
+		disconnectTime := it.FutureNow(1)
 		retry.Run(t, func(r *retry.R) {
 			status, ok := srv.StreamStatus(peerID)
 			require.True(r, ok)
-			require.False(r, status.Connected)
+			require.Equal(r, false, status.Connected)
```

Comment:
nit: Why `require.False -> require.Equal` here?

Reply:
Can change for sure. I used require.Equal because I was doing a bunch of field comparisons so just had them all grouped as require.Equals.